src/Symfony/Bridge/Twig/DataCollector/TwigDataCollector.php

```diff
@@ -15,6 +15,10 @@
 use Symfony\Component\HttpKernel\DataCollector\LateDataCollectorInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
+use Twig\Markup;
+use Twig\Profiler\Dumper\HtmlDumper;
+use Twig\Profiler\Profile;
+use Twig\Source;
```

Comment:
The `NoUnusedImportsFixer` is quite permissive (not removing an unsed import in better than removing an used one) particularly it check if the class in not referenced in a comment. In this case it think it's the "source" word in the license header comment that make the fixer think that the class is still used.